From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Tue, 30 May 2023 18:29:57 +0100
Subject: dt-bindings: media: imx258: Rename to include vendor prefix

imx258.yaml doesn't include the vendor prefix of sony, so
rename to add it.
Update the id entry and MAINTAINERS to match.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
Acked-by: Conor Dooley <conor.dooley@microchip.com>
---
 .../devicetree/bindings/media/i2c/imx258.yaml      | 134 ---------------------
 .../devicetree/bindings/media/i2c/sony,imx258.yaml | 134 +++++++++++++++++++++
 MAINTAINERS                                        |   2 +-
 3 files changed, 135 insertions(+), 135 deletions(-)
 delete mode 100644 Documentation/devicetree/bindings/media/i2c/imx258.yaml
 create mode 100644 Documentation/devicetree/bindings/media/i2c/sony,imx258.yaml

diff --git a/Documentation/devicetree/bindings/media/i2c/imx258.yaml b/Documentation/devicetree/bindings/media/i2c/imx258.yaml
deleted file mode 100644
index 80d2422..0000000
--- a/Documentation/devicetree/bindings/media/i2c/imx258.yaml
+++ /dev/null
@@ -1,134 +0,0 @@
-# SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
-%YAML 1.2
----
-$id: http://devicetree.org/schemas/media/i2c/imx258.yaml#
-$schema: http://devicetree.org/meta-schemas/core.yaml#
-
-title: Sony IMX258 13 Mpixel CMOS Digital Image Sensor
-
-maintainers:
-  - Krzysztof Kozlowski <krzk@kernel.org>
-
-description: |-
-  IMX258 is a diagonal 5.867mm (Type 1/3.06) 13 Mega-pixel CMOS active pixel
-  type stacked image sensor with a square pixel array of size 4208 x 3120. It
-  is programmable through I2C interface.  Image data is sent through MIPI
-  CSI-2.
-
-properties:
-  compatible:
-    const: sony,imx258
-
-  assigned-clocks: true
-  assigned-clock-parents: true
-  assigned-clock-rates: true
-
-  clocks:
-    description:
-      Clock frequency from 6 to 27 MHz.
-    maxItems: 1
-
-  reg:
-    maxItems: 1
-
-  reset-gpios:
-    description: |-
-      Reference to the GPIO connected to the XCLR pin, if any.
-
-  vana-supply:
-    description:
-      Analog voltage (VANA) supply, 2.7 V
-
-  vdig-supply:
-    description:
-      Digital I/O voltage (VDIG) supply, 1.2 V
-
-  vif-supply:
-    description:
-      Interface voltage (VIF) supply, 1.8 V
-
-  # See ../video-interfaces.txt for more details
-  port:
-    $ref: /schemas/graph.yaml#/$defs/port-base
-    additionalProperties: false
-
-    properties:
-      endpoint:
-        $ref: /schemas/media/video-interfaces.yaml#
-        unevaluatedProperties: false
-
-        properties:
-          data-lanes:
-            oneOf:
-              - items:
-                  - const: 1
-                  - const: 2
-                  - const: 3
-                  - const: 4
-              - items:
-                  - const: 1
-                  - const: 2
-
-          link-frequencies: true
-
-        required:
-          - data-lanes
-          - link-frequencies
-
-required:
-  - compatible
-  - reg
-  - port
-
-additionalProperties: false
-
-examples:
-  - |
-    i2c {
-        #address-cells = <1>;
-        #size-cells = <0>;
-
-        sensor@6c {
-            compatible = "sony,imx258";
-            reg = <0x6c>;
-            clocks = <&imx258_clk>;
-
-            port {
-                endpoint {
-                    remote-endpoint = <&csi1_ep>;
-                    data-lanes = <1 2 3 4>;
-                    link-frequencies = /bits/ 64 <320000000>;
-                };
-            };
-        };
-    };
-
-    /* Oscillator on the camera board */
-    imx258_clk: clk {
-        compatible = "fixed-clock";
-        #clock-cells = <0>;
-        clock-frequency = <19200000>;
-    };
-
-  - |
-    i2c {
-        #address-cells = <1>;
-        #size-cells = <0>;
-
-        sensor@6c {
-            compatible = "sony,imx258";
-            reg = <0x6c>;
-            clocks = <&imx258_clk>;
-
-            assigned-clocks = <&imx258_clk>;
-            assigned-clock-rates = <19200000>;
-
-            port {
-                endpoint {
-                    remote-endpoint = <&csi1_ep>;
-                    data-lanes = <1 2 3 4>;
-                    link-frequencies = /bits/ 64 <633600000>;
-                };
-            };
-        };
-    };
diff --git a/Documentation/devicetree/bindings/media/i2c/sony,imx258.yaml b/Documentation/devicetree/bindings/media/i2c/sony,imx258.yaml
new file mode 100644
index 0000000..bee61a4
--- /dev/null
+++ b/Documentation/devicetree/bindings/media/i2c/sony,imx258.yaml
@@ -0,0 +1,134 @@
+# SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
+%YAML 1.2
+---
+$id: http://devicetree.org/schemas/media/i2c/sony,imx258.yaml#
+$schema: http://devicetree.org/meta-schemas/core.yaml#
+
+title: Sony IMX258 13 Mpixel CMOS Digital Image Sensor
+
+maintainers:
+  - Krzysztof Kozlowski <krzk@kernel.org>
+
+description: |-
+  IMX258 is a diagonal 5.867mm (Type 1/3.06) 13 Mega-pixel CMOS active pixel
+  type stacked image sensor with a square pixel array of size 4208 x 3120. It
+  is programmable through I2C interface.  Image data is sent through MIPI
+  CSI-2.
+
+properties:
+  compatible:
+    const: sony,imx258
+
+  assigned-clocks: true
+  assigned-clock-parents: true
+  assigned-clock-rates: true
+
+  clocks:
+    description:
+      Clock frequency from 6 to 27 MHz.
+    maxItems: 1
+
+  reg:
+    maxItems: 1
+
+  reset-gpios:
+    description: |-
+      Reference to the GPIO connected to the XCLR pin, if any.
+
+  vana-supply:
+    description:
+      Analog voltage (VANA) supply, 2.7 V
+
+  vdig-supply:
+    description:
+      Digital I/O voltage (VDIG) supply, 1.2 V
+
+  vif-supply:
+    description:
+      Interface voltage (VIF) supply, 1.8 V
+
+  # See ../video-interfaces.txt for more details
+  port:
+    $ref: /schemas/graph.yaml#/$defs/port-base
+    additionalProperties: false
+
+    properties:
+      endpoint:
+        $ref: /schemas/media/video-interfaces.yaml#
+        unevaluatedProperties: false
+
+        properties:
+          data-lanes:
+            oneOf:
+              - items:
+                  - const: 1
+                  - const: 2
+                  - const: 3
+                  - const: 4
+              - items:
+                  - const: 1
+                  - const: 2
+
+          link-frequencies: true
+
+        required:
+          - data-lanes
+          - link-frequencies
+
+required:
+  - compatible
+  - reg
+  - port
+
+additionalProperties: false
+
+examples:
+  - |
+    i2c {
+        #address-cells = <1>;
+        #size-cells = <0>;
+
+        sensor@6c {
+            compatible = "sony,imx258";
+            reg = <0x6c>;
+            clocks = <&imx258_clk>;
+
+            port {
+                endpoint {
+                    remote-endpoint = <&csi1_ep>;
+                    data-lanes = <1 2 3 4>;
+                    link-frequencies = /bits/ 64 <320000000>;
+                };
+            };
+        };
+    };
+
+    /* Oscillator on the camera board */
+    imx258_clk: clk {
+        compatible = "fixed-clock";
+        #clock-cells = <0>;
+        clock-frequency = <19200000>;
+    };
+
+  - |
+    i2c {
+        #address-cells = <1>;
+        #size-cells = <0>;
+
+        sensor@6c {
+            compatible = "sony,imx258";
+            reg = <0x6c>;
+            clocks = <&imx258_clk>;
+
+            assigned-clocks = <&imx258_clk>;
+            assigned-clock-rates = <19200000>;
+
+            port {
+                endpoint {
+                    remote-endpoint = <&csi1_ep>;
+                    data-lanes = <1 2 3 4>;
+                    link-frequencies = /bits/ 64 <633600000>;
+                };
+            };
+        };
+    };
diff --git a/MAINTAINERS b/MAINTAINERS
index 40312bb..4d93c95 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -19963,7 +19963,7 @@ M:	Sakari Ailus <sakari.ailus@linux.intel.com>
 L:	linux-media@vger.kernel.org
 S:	Maintained
 T:	git git://linuxtv.org/media_tree.git
-F:	Documentation/devicetree/bindings/media/i2c/imx258.yaml
+F:	Documentation/devicetree/bindings/media/i2c/sony,imx258.yaml
 F:	drivers/media/i2c/imx258.c
 
 SONY IMX274 SENSOR DRIVER
